scripts/THALES_migration/stakingEscrowMigration/executeStakingAndEscrowMigration.js

```diff
@@ -66,23 +70,36 @@ async function executeStakingAndEscrowMigration() {
 			continue;
 		}
 
+		if (processedWallets.includes(migratedStakerOrEscrower.wallet)) {
+			console.log('Skipping ' + migratedStakerOrEscrower.wallet + ' as it was already processed');
+			continue;
+		}
 		//send directly if not a staker
-		console.log('Processing migratedStakerOrEscrower ' + migratedStakerOrEscrower);
+		console.log('Processing migratedStakerOrEscrower ' + migratedStakerOrEscrower.wallet);
 		if (migratedStakerOrEscrower.totalStaked == 0) {
-			await thales.transfer(
+			console.log('Sending THALES directly to  ' + migratedStakerOrEscrower.wallet);
+			let tx = await thales.transfer(
```

Comment:
Here there is no check as in line 92
`if (migratedStakerOrEscrower.totalEscrowed === undefined) {
				escrowed = BigNumber.from(0);
			} else {
				escrowed = BigNumber.from(migratedStakerOrEscrower.totalEscrowed);
			}`

Reply:
in line 92 we have to use a BigNumber because we are doing addition later (adding staked to escrowed)